### PR TITLE
Add preexisting conditions to CaseForm

### PR DESCRIPTION
**Pull request by allysonjp715**

### CODE REVIEW

**Comment by allysonjp715** [comment]
[Finish new case manual entry page](https://trello.com/c/haOgIswr/246-finish-new-case-manual-entry-page)

File: verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const hasPreexistingConditionsValues = [undefined, 'Yes', 'No'];
+export default function PreexistingConditions(): JSX.Element {
+    const { values, initialValues } = useFormikContext<CaseFormValues>();
+    return (
+        <Scroll.Element name="preexistingConditions">
+            <fieldset>
+                <legend>Pre-existing conditions</legend>
+                <SelectField
+                    name={`hasPreexistingConditions`}
```
Comment (attwad) [inline verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx:16]: just "hasPreexistingConditions", no need for JS here

File: verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    label="Has preexisting conditions"
+                    values={hasPreexistingConditionsValues}
+                ></SelectField>
+                {values.hasPreexistingConditions === 'Yes' && (
+                    <FormikAutocomplete
+                        name="preexistingConditions"
+                        label="Preexisting conditions"
+                        initialValue={initialValues.preexistingConditions}
+                        multiple={true}
```
Comment (attwad) [inline verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx:25]: just multiple

File: verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx]: Done

File: verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx]: Done

**Pull request merged**